FSJ_django20_project/FSJ/templates/FSJ/coord_student_list.html

```diff
@@ -77,13 +79,22 @@
                         <td>{{ student.last_name }}</td>
                         <td>{{ student.year }}</td>
                         <td>{{ student.program }}</td>
+                        <td>
+                            {% if student.gpa %}
+                              {{ student.gpa }}  
+                            {% else %}
```

Comment:
Doesn't the default behaviour of student.gpa do the same thing as your else statement here (since we're not providing additional content in the event the gpa is blank, the result would still be an empty string?)

Reply:
If we do not provide a student GPA the default is None and that's what shows up in the table, which makes it harder to read GPAs/see which students do or don't have them